pkgs/applications/version-management/git-and-tools/pass-git-helper/default.nix

```diff
@@ -0,0 +1,24 @@
+{ stdenv, buildPythonPackage, isPy3k, fetchFromGitHub, pyxdg }:
+
+buildPythonPackage rec {
+  pname   = "git-pass-helper";
```

Comment:
pass-git-helper instead of git-pass-helper?

Reply:
Sure, you are right.